Dockerfile

```diff
@@ -32,19 +33,19 @@ RUN mkdir -p /opt/QuickAnnotator
 WORKDIR /opt/QuickAnnotator
 COPY . /opt/QuickAnnotator
 
-ENV PATH="/opt/QuickAnnotator/venv/bin:$PATH"
-
+ENV PATH="/opt/venv/bin:$PATH"
 
-RUN python3 -m venv venv \
-    && python3 -m pip install --upgrade pip \
-    && pip install -e .
+RUN python3 -m venv /opt/venv \
```

Comment:
question - if we have a docker container which is already "isolated", what is the added value of as well using a python venv instead of installing in the "base docker o/s"? i always do the latter but am now wondering if that isn't a good practice?

Reply:
It's good to have for development, and (I don't think) causes any performance degradation. 

But I must admit that it's mostly there because of previous Histotools dockerfiles.

Reply:
hmm okay... i agree no performance degradation imparted by it.  i just always think that each additional piece adds complexity and opportunity for things to go astray. if you wanted to remove it, i'd be fine with it

Reply:
Okay. I think I'm fine with it staying. The advantage of venv is that we're a little less coupled to the base docker image - i.e., if we change the base image it's possible that the default python environment changes. With a virtual environment we get a fresh environment every time.

Reply:
smart. makes sense to me, especially given the headaches i had last night with docker : )  agree with you - lets keep it!